components/DisplaySettings/DisplaySettings.styled.js

```diff
@@ -0,0 +1,25 @@
+import styled from "styled-components";
+
+export const SettingsHeader = styled.h2`
+  font-size: 1.1rem;
+  margin: 0;
+  padding: 10px;
+  background: rgba(169, 169, 169, 0.535);
+`;
+
+export const DisplaySettingsContainer = styled.div`
+  display: grid;
+  grid-template-columns: 10fr 1fr 1fr;
+  grid-gap: 10px;
+  padding: 10px;
+`;
+
+export const CenteredHeader2 = styled.h2`
+  text-align: center;
+  text-decoration: underline;
+`;
+
+export const CenteredHeader1 = styled.h1`
```

Comment:
I think the naming is a little confusing here. We have a `header` html tag, but these (and some above are `h1` or `h1`. 

A better name would probably have `heading` in there somewhere.